tests/functional/modules/test_zos_backup_restore.py

```diff
@@ -411,17 +407,17 @@ def test_backup_and_restore_of_data_set_when_restore_location_exists(
 @pytest.mark.parametrize(
     "data_set_include",
     [
-        [DATA_SET_NAME, DATA_SET_NAME2],
-        DATA_SET_PATTERN,
+        ["USER.PRIVATE.TESTDS", "USER.PRIVATE.TESTDS2"],
+        "USER.PRIVATE.*",
     ],
 )
 def test_backup_and_restore_of_multiple_data_sets(ansible_zos_module, data_set_include):
     hosts = ansible_zos_module
+    DATA_SET_NAME = "USER.PRIVATE.TESTDS"
```

Comment:
Why do we keep hardcoded content in this specific test case? If is to make sure those don't exist we can use zos_data_set replace option right?